docker/Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# -*- mode: dockerfile -*-
+# part of the dockerfile to install the python binding
+
+COPY install/python.sh install/
+RUN install/python.sh
+RUN pip3 install jupyter
+ENV PYTHONPATH=/mxnet/python
```

Comment:
I have just added installation of jupyter to the existing dockerfile. Basically I want to be able to have an installed jupyter on after running docker. added the installation to the same file that python was installed